release/gcb/docker_tag_push_lib.sh

```diff
@@ -127,15 +124,29 @@ function docker_push_images() {
   add_docker_creds "${DST_HUB}"
 
   for TAR_PATH in "${OUT_PATH}"/docker/*.tar.gz; do
-    BASE_NAME=$(basename "$TAR_PATH")
-    TAR_NAME="${BASE_NAME%.*}"
-    IMAGE_NAME="${TAR_NAME%.*}"
-
     # if no docker/ directory or directory has no tar files
-    if [[ "${IMAGE_NAME}" == "*" ]]; then
+    if [[ "${TAR_PATH}" == "${OUT_PATH}/docker/*.tar.gz" ]]; then
       break
     fi
+    set_image_vars "$TAR_PATH"
+
     docker load -i "${TAR_PATH}"
-    docker push    "${DST_HUB}/${IMAGE_NAME}:${DST_TAG}"
+
+    docker push "${DST_HUB}/${IMAGE_NAME}:${DST_TAG}${VARIANT_NAME}"
   done
 }
+
+function set_image_vars() {
+  local TAR_PATH=$1
+  BASE_NAME=$(basename "$TAR_PATH")
+  TAR_NAME="${BASE_NAME%.*}"
+  IMAGE_NAME="${TAR_NAME%.*}"
+  VARIANT_NAME=""
+  #check if it is a build variant (e.g. distroless)
+  case "${IMAGE_NAME}" in
+    *-distroless)
+      VARIANT_NAME="-${IMAGE_NAME##*-}"
```

Comment:
Are you adding image name to tag? 
Can you add comments explaining what is expected?

Reply:
Done